test/system/250-systemd.bats

```diff
@@ -292,4 +292,62 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     run_podman network rm -f $netname
 }
 
+# NOTE - running the test locally requires the `podman-play-kube@.service` file
+# **and** the `podman` binary to be installed.
+@test "podman-play-kube@.service template" {
+    skip_if_remote "systemd units do not work with remote clients"
+
+    # Create the YAMl file
+yaml="
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
+spec:
+  containers:
+  - command:
+    - top
+    image: $IMAGE
+    name: test
+    resources: {}
+"
+    yaml_source="$PODMAN_TMPDIR/test.yaml"
+    echo "$yaml" > $yaml_source
+
+    # Dispatch the YAML file
+    systemctl start podman-play-kube@$(systemd-escape $yaml_source).service
+    systemctl is-active podman-play-kube@$(systemd-escape $yaml_source).service
+
+    # The name of the service container is predictable: the first 12 characters
+    # of the hash of the YAML file followed by the "-service" suffix
+    service_container="352a88685060-service"
```

Comment:
I fixed this in my rework, but just a note for next time: you can't do this. The yaml file includes `$IMAGE`, which is a moving target subject to change any time.

Reply:
Need the same change for 700-play.bats